open_event/api/sessions.py

```diff
@@ -146,6 +149,13 @@ def update(self, event_id, service_id, data):
         return obj
 
     def create(self, event_id, data, url):
+        session_custom = json.loads(DataGetter.get_custom_form_elements(event_id).first().session_form)
+        for key in SESSION_POST:
+            if key in session_custom:
+                if session_custom[key]['require'] == 1:
+                    SESSION_POST[key].required = True
+                elif session_custom[key]['require'] == 0:
```

Comment:
@SaptakS you can clone the model in the function. Btw, this just works so forget what I said.